src/scanner.c

```diff
@@ -1,18 +1,46 @@
+#include <string.h>
 #include <tree_sitter/parser.h>
 #include <wctype.h>
 
 enum TokenType {
   STRING_CONTENT,
-  RAW_STRING_LITERAL,
+  RAW_STRING_LITERAL_OPEN,
+  RAW_STRING_LITERAL_CONTENT,
+  RAW_STRING_LITERAL_CLOSE,
```

Comment:
Maybe call the delimiters START/END? Unless the official grammar calls them that

Reply:
Closest I could find was https://github.com/rust-lang/wg-grammar/ which just calls it hashes, doesn't matter